projects/cds/src/cds-models/cds-strategy-request.model.ts

```diff
@@ -4,7 +4,7 @@
  * context
  */
 export interface StrategyRequest {
-  productId?: string;
+  products?: string[];
```

Comment:
change to productCodes

Reply:
Unfortunately we have no flexibility over this, this is what the strategy-service expects as the query parameter so it has to match

Reply:
If it will complicate things, we can just put a comment above this key/property to say like

// this key/property is productCodes in OCC